examples/MethodTest.php

```diff
@@ -107,4 +115,41 @@ public function testWithBody()
 
         $this->assertExpectedCode($generatedCode);
     }
+
+    /**
+     * @expectedException \Gnugat\Medio\Exception\DomainException
+     */
+    public function testSetAbstractAndBody()
+    {
+      Method::make('method')->makeAbstract()->setBody('test');
+    }
+  
+    public function testAbstract()
+    {
+        $method = Method::make('method')->makeAbstract();
+        $this->assertTrue($method->isAbstract());
+
+        $generatedCode = $this->prettyPrinter->generateCode($method);
+        $this->assertExpectedCode($generatedCode);
```

Comment:
Since it generates a single line, we don't need to create a fixture file, we can simply use `$this->assertSame('    public abstract function method();', $generatedCode);`